### PR TITLE
Andy feature/services

### PR DESCRIPTION
**Pull request by jjohnson509**

### CODE REVIEW

**Review by jjohnson509 — COMMENTED** [review]
No conflicts

File: cs4360/libs/crm/nest/dao/src/lib/entities/contact.entity.ts [hunk 1]
```diff
@@ -2,31 +2,43 @@ import {
...
-import {EmployeeEntity} from "./employee.entity";
 
-@Entity("contacts")
+
+import {EmailEntity} from "./email.entity";
+import {PhoneEntity} from "./phone.entity";
+import {UserEntity} from "./user.entity";
+
+@Entity("contact")
 export class ContactEntity {
```
Comment (Newms226) [inline cs4360/libs/crm/nest/dao/src/lib/entities/contact.entity.ts:14]: implement DTO

File: cs4360/libs/crm/shared/src/lib/dto/user.dto.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import {UserTypeDTO} from "./user-type.dto";
+import {ContactDTO} from "@crm/crm/shared";
+
+export interface UserDTO{
+
+  id: string
+
+  username: string;
+  hash: string;
```
Comment (Newms226) [inline cs4360/libs/crm/shared/src/lib/dto/user.dto.ts:9]: hash should not be exposed in a DTO

File: cs4360/ormconfig.json [hunk 3]
```diff
@@ -1,14 +1,19 @@
 {
```
Comment (Newms226) [inline cs4360/ormconfig.json:1]: Did we ever figure out if we need both of these? Both this file and the configuration in the API Module?

File: cs4360/src/libs/crm/nest/dao/src/lib/entities/phone/phone.module.ts [hunk 4]
```diff
@@ -0,0 +1,4 @@
+import { Module } from '@nestjs/common';
```
Comment (Newms226) [inline cs4360/src/libs/crm/nest/dao/src/lib/entities/phone/phone.module.ts:1]: This is an error, this happens when NX doesn't know exactly where to put something.

cd cs4360
rm -fr ./src

**Review by Newms226 — CHANGES_REQUESTED** [review]
Looks Fantastic! I like the addition of a user type, that was smart!

I made a few comments, but looks good after that!

**Pull request closed without merging**